src/backend/distributed/commands/README.md

```diff
@@ -0,0 +1,46 @@
+# Commands
+
+The commands module is modeled after `backend/commands` from the postgres repository and contains the logic for Citus on how to run these commands on distributed objects. Even though the structure of the directory has some resemblence to its postgres relative files here are somewhat more finegrained. This is due to the nature of citus commands that are heavily focused on distributed tables. Instead of having all commands in `tablecmds.c` they are often moved to files that are named after the command.
+
+| File                         | Description |
+|------------------------------|-------------|
+| `create_distributed_table.c` | Implementation of UDF's for creating distributed tables |
+| `drop_distributed_table.c`   | Implementation for dropping metadata for partitions of distributed tables |
+| `extension.c`                | Implementation of `CREATE EXTENSION` commands for citus specific checks |
+| `foreign_constraint.c`       | Implementation of and helper functions for foreign key constraints |
+| `grant.c`                    | Placeholder for code granting users access to relations, implemented as enterprise feature |
+| `indexcmds.c`                | Implementation of commands specific to indices on distributed tables |
```

Comment:
some files have postfix `cmds` some do not. Shall we have a standard on this ?

Reply:
Yeah, I started with making parallels to how the postgres repository is laid out. I already started to divert as most of what we do is related to tables but often would be better suited to be in their own files.

I will change the files with the `cmds` postfix to be different in a way that covers what can be expected inside.